app/src/components/Map/DrawingManager.jsx

```diff
@@ -0,0 +1,109 @@
+import PropTypes from 'prop-types';
+import { Component } from 'preact';
+
+const POLYGON_OPTIONS = {
```

Comment:
these should probably go on app/src/constants.js (at least part of them, like the color and such)